mappings/net/minecraft/client/options/ExponentialGameOption.mapping

```diff
@@ -0,0 +1 @@
+CLASS cva net/minecraft/client/options/ExponentialGameOption
```

Comment:
???

Reply:
It's used by the new scroll sensitivity slider in the mouse settings. While all other sliders in option menus are linear, this slider uses an exponential/logarithmic scale. Roughly the first three quarters are 0 to 1, then it goes up to 10.

No idea how else to call this.

Reply:
I think it's called a "[logarithmic scale](https://en.wikipedia.org/wiki/Logarithmic_scale)", I've never heard of "exponential scale". So I think `LogarithmicGameOption` would be better.

Reply:
It also uses Math.log(), so I changed it now.